### PR TITLE
Extend russian translations & add py.typed for strict mypy

### PR DESCRIPTION
**Pull request by niqzart**
Hi! I'd like to use this lib in one of my projects, but I needed some of the missing translations and the py.typed file (for [strict mypy](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker)). So here is the PR to add all of that & a bit more translations from the ones I could understand. Let me know if any fixes (version bump?) are needed here

### CODE REVIEW

File: locales/ru.po [hunk 1]
```diff
@@ -65,7 +65,7 @@ msgstr "значение должно быть корректным IPv4 адр
 
 #: pydantic_translations/_messages.py:31
 msgid "value is not a valid IPv4 interface"
-msgstr ""
+msgstr "значение должно быть корректным IPv4 интерфесом"
```
Comment (sshishov) [inline locales/ru.po]:
```suggestion
msgstr "значение должно быть корректным IPv4 интерфейсом"
```

File: locales/ru.po [hunk 2]
```diff
@@ -77,7 +77,7 @@ msgstr "значение должно быть корректным IPv4 или
 
 #: pydantic_translations/_messages.py:34
 msgid "value is not a valid IPv4 or IPv6 interface"
-msgstr ""
+msgstr "значение должно быть корректным IPv4 или IPv6 интерфесом"
```
Comment (sshishov) [inline locales/ru.po]:
```suggestion
msgstr "значение должно быть корректным IPv4 или IPv6 интерфейсом"
```

File: locales/ru.po [hunk 3]
```diff
@@ -89,7 +89,7 @@ msgstr "значение должно быть корректным IPv6 адр
 
 #: pydantic_translations/_messages.py:37
 msgid "value is not a valid IPv6 interface"
-msgstr ""
+msgstr "значение должно быть корректным IPv6 интерфесом"
```
Comment (sshishov) [inline locales/ru.po]:
```suggestion
msgstr "значение должно быть корректным IPv6 интерфейсом"
```

File: locales/ru.po [hunk 4]
```diff
@@ -371,27 +371,27 @@ msgstr "некорректная длина кортежа {actual_length}, ож
 #: pydantic_translations/_messages.py:115
 #, python-brace-format
 msgid "URL invalid, extra characters found after valid URL: {extra!r}"
-msgstr ""
+msgstr "URL невалиден, после валидного URL находятся дополнительные символы: {extra!r}"
```
Comment (sshishov) [inline locales/ru.po]: In Russian language there is no such word `невалиден`, I would change it everywhere to `недопустимый`
```suggestion
msgstr "недопустимый URL, после допустимого URL находятся дополнительные символы: {extra!r}"
```

File: locales/ru.po [hunk 5]
```diff
@@ -371,27 +371,27 @@ msgstr "некорректная длина кортежа {actual_length}, ож
...
 
 #: pydantic_translations/_messages.py:116
 msgid "URL host invalid, top level domain required"
-msgstr ""
+msgstr "некорректный URL хост, домен верхнего уровня должен быть указан"
 
 #: pydantic_translations/_messages.py:117
 msgid "URL port invalid, port cannot exceed 65535"
-msgstr "некорретный URL порт, номер порта должен быть не больше 65535"
+msgstr "некорректный URL порт, номер порта должен быть не больше 65535"
```
Comment (sshishov) [inline locales/ru.po]: For instance here also:
```suggestion
msgstr "недопустимый URL порт, номер порта должен быть не больше 65535"
```

File: locales/ru.po [hunk 6]
```diff
@@ -371,27 +371,27 @@ msgstr "некорректная длина кортежа {actual_length}, ож
...
 
 #: pydantic_translations/_messages.py:118
 msgid "invalid or missing URL scheme"
 msgstr "URL должен содержать корректную схему"
 
 #: pydantic_translations/_messages.py:118
 msgid "URL scheme not permitted"
-msgstr ""
+msgstr "URL схема запрещена"
```
Comment (sshishov) [inline locales/ru.po]: not permitted = `не разрешен`
forbidded = `запрещен`
```suggestion
msgstr "указание URL схемы не разрешено"
```

**Review by sshishov — COMMENTED** [review]
Overall looks good, but I would use correct Russian words instead of naturalized computer engineer's slang

**Comment by orsinium** [comment]
Thank you! I like the suggestions from @sshishov, the suggested translations sound a bit more natural to me.

**Comment by niqzart** [comment]
Yes, they do make sense if you want less slang (and it seems to be the goal for this lib). I've added all of them into the branch, thanks!

**Comment by orsinium** [comment]
Thank you!

**Pull request merged**